src/printer.js

```diff
@@ -1941,7 +1948,7 @@ function genericPrintNoParens(path, options, print) {
   }
 }
 
-function printStatementSequence(path, options, print) {
+function printStatementSequence(path, options, print, isClass) {
```

Comment:
While it's technically "more correct", I think I'm going to get rid of this `isClass` thing because it's not necessary and stinks a bit